src/examples/zkapps/dex/run.ts

```diff
@@ -9,9 +16,9 @@ import {
 } from './dex.js';
 
```

Comment:
Was there a reason you created a separate `arbitrary_token_interaction.ts` file and this `run.ts` file? Is the goal to only execute the `run.ts` file in CI and not the `arbitrary_token_interaction.ts` file?

Reply:
Ok that makes sense. Maybe we should document the file with this info so we don't forget later on.